index.js

```diff
@@ -1,22 +1,24 @@
-const { createStore } = require('redux')
-const { Set } = require('immutable')
 const assert = require('assert')
+const Immutable = require('immutable')
+
```

Comment:
No need to separate them with a blank line (keep this separation for global vs local imports), just order them.